### DISCUSSION TITLE
Cannot create more than 5 automatic discount with discountAutomaticAppCreate

### DISCUSSION THREAD

**Discussion by peter-visualsquares** [Shopify/function-examples#198] [Feedback: Discounts APIs]
From doc at most 25 automatic discounts can be active on a Shopify store at the same time. However when I use discountAutomaticAppCreate to create automatic discounts powered by Shopify functions, I can create at most 5 automatic discounts. Beyond that I will see below error msg
```
The active period overlaps with that of another price rule
```
It doesn't align with the limit mentioned in the doc.

Ref:
1. https://help.shopify.com/en/manual/discounts/combining-discounts

**Comment by cyberabis** [comment]
Is there any other API to create discount? We are also facing this issue.

**Reply by gpisarski** [reply]
Any news when this limit will be increased?

**Reply by sergiop14** [reply]
Based on the Shopify discord discussion:

`we'll be looking into this limitation in the 2nd half of the year. I suggest to also add to this thread: https://github.com/Shopify/function-examples/discussions/198. We'll communicate updates there.`

I hope they remove this limitation as most users need more than 5 active discounts at a time.

**Comment by cyberabis** [comment]
Is there any update here... it was originally announced we can use 25 automatic discounts. But not it's max 5 across all apps.
We also see a loophole here. We can reactivate expired automatic discounts and cross the 5 limit.

**Reply by sergiop14** [reply]
I've tried it but only activate 5 discounts. And if I try to activate it through the API, it returns the common error. Can you share the process? Thanks

**Comment by thosakwe** [comment]
This will make a huge impact for apps that give different discounts to customers depending on the buyer identity.

Many merchants have several loyalty program tiers, for example, so they run into this issue frequently.
